__tests__/app.test.js

```diff
@@ -99,3 +110,29 @@ describe("GET /api/articles/:article_id", () => {
             });
     });
 });
+
+describe("GET /api/articles", () => {
+    test("200: should respond with an articles array containing all article objects", () => {
```

Comment:
My only comment is to either change the test description to show that you are also testing for how the articles are sorted, or you can test how they're sorted in a different test block.